js/components/url.js

```diff
@@ -9,11 +9,22 @@ Fliplet.FormBuilder.field('url', {
       type: String
     }
   },
+  data: function() {
+    return {
+      rules: {
+        urlCase: new RegExp(/^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@%!\$&'\*\+,;=.]+$/i)
+      }
+    };
+  },
   validations: function() {
     var rules = {
       value: {
         // URL regex taken form https://www.regextester.com/94502 and added % sign
-        url: window.validators.helpers.regex('', /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@%!\$&'\*\+,;=.]+$/i)
+        url: function(value) {
+          value = value.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
```

Comment:
Please add a comment referencing https://stackoverflow.com/questions/990904/remove-accents-diacritics-in-a-string-in-javascript